src/leaderboardBot.py

```diff
@@ -203,6 +203,39 @@ def getLeaderboardThreshold(self, rank=200):
 
         return dict
 
+    def get_leaderboard_range(self, start_rank, end_rank):
```

Comment:
I'm using snake case here because it is pythonic. This is a deviation from the rest of the class.